p2p/message_test.go

```diff
@@ -601,3 +602,31 @@ func TestDecodeTransactionMessageTwoExtrinsics(t *testing.T) {
 		t.Fatalf("Fail: got: %v expected %v", *decodedMessage, expected)
 	}
 }
+
+func TestDecodeConsensusMessage(t *testing.T) {
+	encMsg, err := common.HexToBytes("0x080000000000000000")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	buf := &bytes.Buffer{}
+	buf.Write(encMsg)
+
+	m := new(ConsensusMessage)
+	err = m.Decode(buf)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	ID := uint32(8)
+	encId := make([]byte, 8)
+	binary.LittleEndian.PutUint32(encId, ID)
+	expected := &ConsensusMessage{
+		ID:   ID,
+		Data: []byte{0, 0, 0, 0, 0},
```

Comment:
Perhaps use values other than zero, I think it would create a more substantial test case

Reply:
cool, thanks for the hint, could you please check again after the update ?